Sources/IBLinterKit/Extensions/Config+LintablePaths.swift

```diff
@@ -7,14 +7,64 @@
 
 import Foundation
 
+private struct InterfaceBuilderFiles {
+    var xib: Set<URL> = []
+    var storyboard: Set<URL> = []
+}
+
 extension Config {
-    public func lintablePaths(workDirectory: URL, fileExtension: String, fileManager: FileManager = .default) -> [URL] {
-        let paths: Set<URL> = self.included.isEmpty ? glob(pattern: "\(workDirectory.path)/**/*.\(fileExtension)") : []
-        let excluded: [URL] = self.excluded.flatMap { glob(pattern: workDirectory.appendingPathComponent("\($0)/**/*.\(fileExtension)").path) }
-        let included: [URL] = self.included.flatMap { glob(pattern: workDirectory.appendingPathComponent("\($0)/**/*.\(fileExtension)").path) }
-        let lintablePaths: [URL] = (paths + included).filter { (path: URL) -> Bool in
-            return !excluded.contains { $0.absoluteString == path.absoluteString }
+    public func lintablePaths(workDirectory: URL, fileManager: FileManager = .default) -> (xib: Set<URL>, storyboard: Set<URL>) {
```

Comment:
I think it seems to be a bit strange what implementing this feature as Config's extension.

How about implementing this on `Validator` directly?

Reply:
that makes sense 👍